db-connector/src/main/java/uk/nhs/adaptors/connector/dao/ImmunizationSnomedCTDao.java

```diff
@@ -9,7 +9,7 @@
 public interface ImmunizationSnomedCTDao {
     @SqlQuery("select_immunization_concept_id")
     @UseClasspathSqlLocator
-    ImmunizationSnomedCT getImmunizationSnomednUsingConceptId(@Bind("conceptId") String conceptId);
+    ImmunizationSnomedCT getImmunizationSnomedUsingConceptOrDescriptionId(@Bind("sctid") String sctId);
```

Comment:
Worth noting too that the return type of `ImmunizationSnomedCT` is completely useless, this method is only called within `DatabaseImmunizationChecker`, in turn only returning a `boolean`. Why not just have this return a `boolean`?

Again, technically **out of scope** of this ticket, though will be tidied in a separate PR.

Reply:
**TODO** another PR.